### PR TITLE
Update DepositionMTK.jl

### PR DESCRIPTION
**Pull request by xiaor2**
Imported the data and finished the equations, but still working on the debugging.

### CODE REVIEW

**Comment by xiaor2** [comment]
recheck

File: src/DepositionMTK.jl [hunk 1]
```diff
@@ -2,4 +2,396 @@ module DepositionMTK
...
+	[inf, inf, inf, inf, 400, 800, inf, inf, inf, inf, inf],
+	[inf, 120, 240, 140, 250, 190, inf, inf, 160, 200, 300]]
+
+# ╔═╡ 9d86d682-92bd-4b26-ae23-c4a385374a7c
+md"""
+r_lu signifies leaf cuticles in healthy vegetation and otherwise the outer surfaces in the upper canopy.
+"""
+
+# ╔═╡ 301eaff9-06c8-4726-9067-27a5e813f86c
+r_lu = [
```
[the remark this answers is not in the document]
Comment (ctessum) [reply src/DepositionMTK.jl:35]: We can also make these data matrixes `const`ants and [StaticArrays](https://github.com/JuliaArrays/StaticArrays.jl) to improve performance:

``` julia
using StaticArrays
const x = SA_F32[ 
      1 2
       3 4
]
```

File: src/DepositionMTK.jl [hunk 2]
```diff
@@ -2,4 +2,396 @@ module DepositionMTK
...
+end
+
+# ╔═╡ adfb7cbf-3dc0-4fa7-9adc-66e203accffc
+md"""
+Properties of various gases from Wesely (1989) Table 2.
+"""
+
+# ╔═╡ 34f744cb-d9fc-44b4-a61e-3262ee2b1e65
+begin
+	So2Data = GasData(1.9, 1.e5, 0)
```
Comment (ctessum) [inline src/DepositionMTK.jl:116]: I think these values can also be constants, e.g.

``` julia
const So2Data = GasData(1.9, 1.e5, 0)
```

File: src/DepositionMTK.jl [hunk 3]
```diff
@@ -2,4 +2,396 @@ module DepositionMTK
...
+	end
+	
+	if rainOrDew
+		rs *= 3
+	end
+	return rs
+end
+
+# ╔═╡ f0481175-a676-4df0-ad1e-aabcfefb0995
+function r_dc(G::Float64, θ::Float64)
```
Comment (ctessum) [inline src/DepositionMTK.jl:201]: To make this more flexible, we can convert all the Float64 values (here and elsewhere) to AbstractFloat types. So:

``` julia
function r_dc(G::T, θ::T) where T<:AbstractFloat
```

etc.

**Review by ctessum — COMMENTED** [review]
Looking good so far! I left several comments.

Also, Julia provides the ability to define the units of values: https://painterqubits.github.io/Unitful.jl/stable/

I think we should try to use that throughout, to provide an additional check that all of the units are correct.

**Comment by ctessum** [comment]
Also you can put "I have reviewed and hereby sign the CLA" in a comment here to sign the CLA, if you are okay with doing so.

**Comment by xiaor2** [comment]
I have reviewed and hereby sign the CLA

**Pull request closed without merging**